src/OmniSharp.Roslyn/OmniSharpWorkspace.cs

```diff
@@ -37,6 +39,11 @@ public OmniSharpWorkspace(HostServicesAggregator aggregator, ILoggerFactory logg
 
         public override bool CanOpenDocuments => true;
 
+        public void AddWaitForProjectModelReadyEventHandler(Func<string, Task> eventHandler)
```

Comment:
It's not really an EventHandler. Is there a better name?

Reply:
@rchande Do you think "CallBack" or just "Hander" sound better?

Reply:
Handler is fine.
I guess the overall concern is that there's now a circular dependency between ProjectLoader and OmniSharpWorkspace. I'm not sure of a better way to improve this right now though...